index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+session_start();
+
+require('functions.php');
+
+$plant_data = getDB ();
+
+DBCheck($plant_data);
```

Comment:
because this function echos inside it, from here it looks like this function does nothing. its a bit odd